Octokit.Tests.Integration/Clients/AssigneesClientTests.cs

```diff
@@ -38,4 +41,22 @@ public AssigneesClientTests()
         var assignees = await _github.Issue.Assignee.GetAllForRepository(_context.RepositoryOwner, _context.RepositoryName);
         Assert.True(assignees.Any(u => u.Login == Helper.UserName));
     }
+
+    [IntegrationTest]
+    public async Task CanAddAssignees()
```

Comment:
Dont forget to add an integration test for `CanRemoveAssignees` method :grinning: